Content.IntegrationTests/Tests/EntityTest.cs

```diff
@@ -40,12 +43,25 @@ public async Task Test()
                 grid = mapLoader.LoadBlueprint(mapId, "Maps/stationstation.yml");
             });
 
+            client.Assert(() =>
```

Comment:
Split this code into a new method within this same class, which you could call `NotAbstractSpriteTest()`, and rename this method to `SpawnTest()`.